src/Microsoft.VisualStudio.Threading.Analyzers.Tests/Microsoft.VisualStudio.Threading.Analyzers.Tests.csproj

```diff
@@ -19,12 +19,16 @@
     <ProjectReference Include="..\Microsoft.VisualStudio.Threading\Microsoft.VisualStudio.Threading.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.3.2" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="14.3.0" />
+    <PackageReference Include="MicroBuild.VisualStudio" Version="$(MicroBuildVersion)" PrivateAssets="all" />
```

Comment:
:memo: It seems the assembly isn't given a strong name (and thus can't be the target of an `InternalsVisibleTo`) unless this is included.

Reply:
True. But this also activates real signing and other quality checks not necessary for unit testing. So if we include this, we should also reference `MicroBuild.NonShipping` to prevent the unnecessary stuff.

That said, I'm curious why we need IVT, because I don't like it. It leads to testing... *internals* 😲 instead of what customers really care about. That produces brittle tests that raise the cost of healthy refactoring, and eventually deleting the tests when they fail since it's harder to tie them to a user scenario.

Reply:
> we should also reference MicroBuild.NonShipping to prevent the unnecessary stuff.

We already did reference this.

> That said, I'm curious why we need IVT ...

Access to the `Descriptor` fields, which improves the ability of the verifier to pick up the default configuration of an analyzer (severity, message format, ID).